packages/webview_flutter/webview_flutter_wkwebview/lib/src/foundation/foundation.dart

```diff
@@ -199,6 +199,20 @@ class NSUrlRequest {
   final Map<String, String> allHttpHeaderFields;
 }
 
+/// A URL load request that is independent of protocol or URL scheme.
+///
+/// Wraps [NSHttpUrlResponse](https://developer.apple.com/documentation/foundation/nshttpurlresponse?language=objc).
+@immutable
+class NSHttpUrlResponse {
+  /// Constructs an [NSHttpUrlResponse].
+  const NSHttpUrlResponse({
+    required this.statusCode,
+  });
+
+  /// The URL being requested.
```

Comment:
I think this was supposed to be 
```suggestion
  /// The response’s HTTP status code.
```